packages/strapi-plugin-upload/admin/src/components/ListWrapper/index.js

```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components';
 
 const ListWrapper = styled.div`
-  margin-top: 2px;
+  margin-top: 20px;
```

Comment:
Don't forget to check if this value match in the content manager modal. If not, you can just create another wrapper for the plugin case and leave the old value by default.

Reply:
you're right